pkg/hack/boilerplate.go.txt

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
Why is this file required?

Reply:
This was required to run codegen locally, but since we are using build image, and we can use the one under codegenerator pkg ``` /go/pkg/mod/k8s.io/code-generator@v0.24.4/hack/boilerplate.go.txt```